lib/metasploit/framework/command/console.rb

```diff
@@ -39,12 +39,50 @@ def spinner
     end
   end
 
+  def start_profiler
+    if ENV['METASPLOIT_CPU_PROFILE']
+      gem 'perftools.rb'
+      require 'perftools'
+
+      formatted_time = Time.now.strftime('%Y%m%d%H%M%S')
+      root = Pathname.new(__FILE__).parent
```

Comment:
I think moving this logic from the root directory to inside `commands` now calculates the wrong root directory, will fix 🤞